packages/core/scss/components/grid/_layout.scss

```diff
@@ -1266,6 +1266,13 @@
                 align-items: center;
             }
         }
+
+        .k-columnmenu-indicators {
+            padding-inline: k-spacing(1);
```

Comment:
Maybe we could define a variable here?

Reply:
I don't want to add ColumnMenu variables at this point as the styles are spread out as it is. I think it would be better if we do this when we are consolidating the styles.